main/main.c

```diff
@@ -131,24 +131,41 @@ VM_API vm_state_t *vm_main_default(void) {
 
 #else
 
+/* Usage:
+ *
+ * minivm [--dis] FILE
+ */
 int main(int argc, const char *argv[argc]) {
   if (argc < 2) {
     for (const char *i = VM_CAN_NOT_RUN; *i != '\0'; i++) {
       vm_putchar(*i);
     }
-    return 0;
+    return 1;
   }
   if (vm_main_str_eq(argv[1], "--dis")) {
+    if (argc < 4) {
+      for (const char *i = VM_CAN_NOT_RUN; *i != '\0'; i++) {
+        vm_putchar(*i);
+      }
+      return 1;
```

Comment:
changed to an error code, but please leave feedback if that's not right.

Reply:
this is not correct